docs/blueprints/EP036.rst

```diff
@@ -0,0 +1,237 @@
+            "napps_table_groups": {"mef_eline": ["epl"]}
+          }
+          4: {
+            "table_miss_flow": {},
+            "napps_table_groups": {"telemetry_int": ["base"]}
+          }
+          5: {
+            "table_miss_flow": {},
+            "napps_table_groups": {}
+          }
+        }
+      }
+    }
+
+Events
+======
+
+Subscribed
+-----------
+
+  - **kytos/flow_manager.flow.added**
+  - **kytos/of_core.handshake.completed**
+  - **kytos/flow_manager.flow.error**
+  - **kytos/[mef_eline|telemetry_int|coloring|of_lldp].enable_table** - Every NApp sends an event with their table settings as content.
+  - **kytos/[mef_eline|telemetry_int|coloring|of_lldp].cookie** - Every NApp sends an event with their cookie information
+
+kytos/<napp_name>.enable_table
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This event is sent from other NApps with information about their table settings.
+
+.. code:: javascript
+  "content": {
+    "mef_eline": {"epl": 3}
+  }
+
+kytos/<napp_name>.cookie
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: javascript
+  "content": {
+    "of_lldp": "0xac00000000000001"
+  }
+
+Published
+---------
+
+kytos/multi_table.enable_table
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This event should sent before any NApp sends its first flow to be published.
+
+.. code:: javascript
+
+  "content": {
+    "mef_eline": {"epl": 3, "evpl": 2},
+    "of_lldp": {"base": 0},
+    "coloring": {"base": 0}
+  }
+
+Endpoint API
+============
+  
+  - GET `/api/kytos/multi_table/v1/pipeline/` - Get all pipelines in the database. Parameters enabled and disabled are allowed.
+
+  - GET `/api/kytos/multi_table/v1/pipeline/<pipeline_id>` - Get a single pipeline that matches the given pipeline id.
+
+  - POST `/api/kytos/multi_table/v1/pipeline/<pipeline_id>/enable` - Enable pipeline to be applied to all switches
+
+  - POST `/api/kytos/multi_table/v1/pipeline/<pipeline_id>/disable` - Disable pipeline in case of the user wants to return to the default set up.
+
+  - POST `/api/kytos/multi_table/v1/pipeline` - Add pipeline to database. Its body is similar to "pipeline.multi_table".
+
+  - DELETE `/api/kytos/multi_table/v1/pipeline/<pipeline_id>` - Delete only disabled pipeline from database otherwise it should prompt the user to disable the pipeline first.
+
+Enabling pipeline
+==================
+
+Im simple terms, enabling a pipeline will mean to modify every flow so it complies with the pipeline configuration. The process to successfully enable a pipeline is as follows:
+
+  1. **of_multi_table** publishes `kytos/multi_table.enable_table` event with pipeline content.
+  2. Wait for all NApp to publish their own events with their respective table settings.
+  3. Disable the previous enabled pipeline, if there was any.
+  4. **of_multi_table** requests all flows from **flow_manager** API.
+  5. **of_multi_table** duplicates every flow that needs a different table with NApps cookie information.
+  6. Insert duplicated flows to the database.

```

Comment:
Can we get rid of step 6? Step 7 mentioning installing flows with a post request to `flow_manager` already implies that they'll be upserted.